### PR TITLE
Footer

### LINKED ISSUE #30: Footer

### PR DESCRIPTION
**Pull request by basil-kawak**
PTAL

### CODE REVIEW

**Comment by basil-kawak** [comment]
this is design 
design
this is my work
my work

File: src/components/common/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import React, { useState } from "react";
+import "./index.css";
+import { Form, Container, Row, Col, Button, Nav } from "react-bootstrap";
+const Footer = () => {
```
Comment (louisrli) [inline src/components/common/Footer.jsx:5]: newline above

File: src/components/common/Footer.jsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  const [email, setEmail] = useState("");
+  const handleSubmit = (e) => {
+    e.preventDefault();
```
Comment (louisrli) [inline src/components/common/Footer.jsx:8]: write something like this, this is how we do it normally: `// TODO(mygithubname): connect this to newsletter`

File: src/components/common/Footer.jsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        <div className="justify-content-center">
+          <h3>SUBSCRIBE TO STAY UP TO DATE</h3>
+          <Form onSubmit={handleSubmit}>
+            <Form.Row className="align-items-center">
+              <Col>
+                <Form.Control
+                  onChange={(e) => setEmail(e.target.value)}
+                  value={email}
+                  className="formControl"
+                  placeholder="Enter Your Email Address"
```
Comment (louisrli) [inline src/components/common/Footer.jsx]: `Your email address`

File: src/components/common/Footer.jsx [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+          <Col md={3}>
+            <h4>Follow Us</h4>
+            <Nav className="socialCircle">
+              <Nav.Link href="#" className="icoLinkedin" title="Linkedin">
+                <i className="fa fa-linkedin"></i>
+              </Nav.Link>
+              <Nav.Link href="#" className="icoFacebook" title="Facebook">
+                <i className="fa fa-facebook"></i>
+              </Nav.Link>
+              <Nav.Link href="#" className="icoTwiter" title="Facebook">
```
Comment (louisrli) [inline src/components/common/Footer.jsx]: `ico` -> `icon` here and everywhere else
typo in twitter

**Review by louisrli — CHANGES_REQUESTED** [review]
generally looking good

**Comment by basil-kawak** [comment]
PTAL

**Review by louisrli — APPROVED** [review]
very solid work basil

**Review by Samah-Mustafa — APPROVED** [review]

**Review by basil-kawak — COMMENTED** [review]
good job

**Pull request merged**